addons/mail/static/src/components/message/message.scss

```diff
@@ -5,7 +5,7 @@
 .o_Message {
     display: flex;
     flex: 0 0 auto;
-    padding: map-get($spacers, 2);
+    padding-right: map-get($spacers, 2);
```

Comment:
This is ok but while we are at it I would just do everything with margin on children then, not a mix of padding and margin.